configure.ac

```diff
@@ -4,20 +4,20 @@ AC_PREREQ([2.60])
 # the API. All changes in experimental modules are treated as
 # backwards-compatible and therefore at most increase the minor version.
 define(_PKG_VERSION_MAJOR, 0)
-define(_PKG_VERSION_MINOR, 1)
-define(_PKG_VERSION_BUILD, 0)
+define(_PKG_VERSION_MINOR, 2)
+define(_PKG_VERSION_PATCH, 1)
 define(_PKG_VERSION_IS_RELEASE, false)
 
 # The library version is based on libtool versioning of the ABI. The set of
 # rules for updating the version can be found here:
 # https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
 # All changes in experimental modules are treated as if they don't affect the
 # interface and therefore only increase the revision.
-define(_LIB_VERSION_CURRENT, 0)
-define(_LIB_VERSION_REVISION, 0)
+define(_LIB_VERSION_CURRENT, 1)
+define(_LIB_VERSION_REVISION, 1)
```

Comment:
Why has `revision` been incremented? Incrementing of `current` implies setting `revision` to zero.

Reply:
Did you notice that `CURRENT` and `REVISION` are incremented in different commits? The idea is that after we create the release commit we immediately increment the package `PATCH` version and library `REVISION` to have sensible version numbers in master that are distinct from the release.

Reply:
> Did you notice that `CURRENT` and `REVISION` are incremented in different commits?

I didn't (

In that case, changes make sense.